internal/clients/team3/clientgeneral.go

```diff
@@ -32,18 +32,26 @@ func NewClient(clientID shared.ClientID) baseclient.Client {
 
 	// Set trust scores
 	for _, islandID := range shared.TeamIDs {
+
+		// Initialise trust scores for all islands except our own
+		if islandID == shared.Team3 {
```

Comment:
use  `p.c.BaseClient.GetID()` instead of shared.Team3, as we want to be able to duplicate this client to the other islands

Reply:
It would be great if you could replace all references to our island with this

Reply:
Done! :)